plugins/Web/WebBackend.vala

```diff
@@ -0,0 +1,176 @@
+// -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
+/*-
+ * Copyright (c) 2013 Maya Developers (https://launchpad.net/maya)
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public
+ * License along with this library; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ * Authored by: Corentin Noël <tintou@mailoo.org>
+ */
```

Comment:
Not 100% sure but I think something more like
```
/*
 * Copyright 2024 elementary, Inc. (https://elementary.io)
 * SPDX-License-Identifier: GPL-3.0-or-later
 *
 * (This is optional:) Authored by: Your Name <your@mail> 
 */
```
is used currently.
Maybe @danirabbit can give some insight here :)

Reply:
Ah yeah we definitely prefer the compact SPDX license header these days. And yeah since it's not clear who "Maya Developers" is (maybe there was a launchpad group at one time, but its inactive) it would be clearer to assign copyright to elementary, Inc. since that's actually a defined entity.

And yeah it looks like this repo is GPL-3.0 but the current plugins are LGPL-2.0. I think that's fine. @tintou knows better about licenses.

And yeah since @tintou didn't author this file I'm sure it would be better to either include yourself as the author here or you can leave that line blank if you don't want to leave a contact :)

Reply:
Yeah I get the idea from this file: https://github.com/elementary/calendar/blob/79a17454772f322e366c70e7f2e9f4b74ccd03dc/plugins/CalDAV/CalDAVBackend.vala